src/ReturnBookControl.java

```diff
@@ -1,77 +1,75 @@
 class ReturnBookControl {
 
-	private ReturnBookUI ui;
-	private enum CONTROL_STATE { INITIALISED, READY, INSPECTING };
-	private CONTROL_STATE state;
+	private ReturnBookUi returnBookUi;
+	private enum controlState { INITIALISED, READY, INSPECTING };
+	private controlState state;
 	
 	private library library;
 	private loan currentLoan;
 	
 
 	ReturnBookControl() {
 		this.library = library.INSTANCE();
-		state = CONTROL_STATE.INITIALISED;
+		state = controlState.INITIALISED;
 	}
 	
 	
-	void setUI(ReturnBookUI ui) {
-		if (!state.equals(CONTROL_STATE.INITIALISED)) {
+	void setUI(ReturnBookUi returnBookUi) {

```

Comment:
should setUI be setUi to conform to the rest of the changes?

Reply:
adjusted